src/index.js

```diff
@@ -53,6 +58,8 @@ define('[tooltip]', {
     // Reset the tooltip element so it can be re-used.
     this.elmTooltip.innerHTML = '';
     this.elmTooltip.style.display = 'none';
+    const customClasses = this.elmTooltip.classList
+    Array.from(customClasses).forEach(cl => this.elmTooltip.classList.remove(cl));
```

Comment:
If I'm reading this right. It gets a list of all the classes on `elmTooltip`. Then converts that list into an array so it can use `forEach()`; loops over each item and then removes the item from the classList.

What if the user has a custom class on the `#pep-tooltips` element that they want to keep? For example, the user writes logic to apply Brand-specific classes to tooltips based on the URL route.  So their element looks something like `<div id="pep-tooltips" class="brand-cheetos"></div>` or `<div id="pep-tooltips" class="brand-mtnDew"></div>`. 

Another question. The attribute, `data-tooltip-class`, is named `class` like the element attribute `<div class="" ></div>`. Does this mean the user could pass a space-separated list of classes to add/remove like the DOM's [native API](https://developer.mozilla.org/en-US/docs/Web/API/Element/className)?


FYI, if you want to nuke all the classes on an element, the fastest way is `elm.className = ''`.

FYI: the [DOMTokenList](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList) returned by [classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) has a `forEach` function, so you don't need the `Array.from()`.